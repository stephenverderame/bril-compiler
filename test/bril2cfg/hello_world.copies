digraph {
	subgraph cluster_main {
		label="main()";
		rankdir="TB";
		main_0 [label="START", shape="rectangle", style="rounded"];
		main_3 [label="[]\n----------\nx := 10\nprint x\n----------\n[]", shape="rectangle", style="rounded"];

		main_0 -> main_3;
	}
}
