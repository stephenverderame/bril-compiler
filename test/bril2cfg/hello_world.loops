digraph {
	subgraph cluster_main {
		label="main()";
		rankdir="TB";
		main_0 [label="START", shape="rectangle", style="rounded"];
		main_3 [label="x := 10\nprint x", shape="rectangle", style="rounded"];

		main_0 -> main_3;
	}
}
