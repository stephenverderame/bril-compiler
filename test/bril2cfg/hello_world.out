digraph {
	subgraph cluster_main {
		label="main";
		rankdir="TB";
		0 [label="Start", shape="ellipse"];
		3 [label="x := 10", shape="ellipse"];
		4 [label="print x", shape="ellipse"];

		0 -> 3;
		3 -> 4;
	}
}
